src/components/FlatListItem/index.js

```diff
@@ -3,31 +3,34 @@ import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
 import Icon from 'react-native-vector-icons/MaterialIcons';
 
+import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
-import {COLORS} from '../../utils/theme';
-import {useDynamicColors} from '../../hooks/useToggleColors';
 
-const FlatListItem = ({value, index, select, onSelect, icon, desc, isAnonimity = false}) => {
-  const dynamicColor = useDynamicColors(isAnonimity);
+const FlatListItem = ({value, index, select, onSelect, icon, desc}) => {
   if (icon) {
     return (
-      <TouchableOpacity testID="onselect" style={styles.container} onPress={() => onSelect(index)}>
+      <TouchableOpacity
+        testID='onselect'
+        style={styles.container}
+        onPress={() => onSelect(index)}>
         <View style={styles.contentText}>
           {icon}
           <View style={styles.containerLabel}>
             <Text style={styles.labelIcon}>{value}</Text>
             <Text style={styles.desc}>{desc}</Text>
           </View>
         </View>
-        {index === select && <Icon name="check-circle" color={dynamicColor.primary} size={20} />}
+        {index === select && (
+          <Icon name="check-circle" color={colors.bondi_blue} size={20} />
+        )}
       </TouchableOpacity>
     );
   }
   return (
-    <TouchableOpacity testID="noicon" style={styles.container} onPress={() => onSelect(index)}>
+    <TouchableOpacity testID='noicon' style={styles.container} onPress={() => onSelect(index)}>
       <Text style={styles.label}>{value}</Text>
       {index === select && (
-        <Icon testID="circle-icon" name="check-circle" color={dynamicColor.primary} size={20} />
+        <Icon testID='circle-icon' name="check-circle" color={colors.bondi_blue} size={20} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk seem to be primarily about replacing the old color handling system with a new one. The `useDynamicColors` hook and `COLORS` import have been replaced with a direct import of `colors`. This change could potentially introduce issues if the colors used are supposed to adapt to certain conditions (like dark mode or user preferences), which was previously handled by `useDynamicColors`. If the new `colors` module doesn't handle these cases, it might be worth considering reintroducing dynamic color handling.

```diff
- import {COLORS} from '../../utils/theme';
- import {useDynamicColors} from '../../hooks/useToggleColors';
+ import {colors} from '../../utils/colors';

...

- const dynamicColor = useDynamicColors(isAnonimity);
...
- <Icon name="check-circle" color={dynamicColor.primary} size={20} />
...
- <Icon testID='circle-icon' name="check-circle" color={dynamicColor.primary} size={20} />
+ <Icon name="check-circle" color={colors.bondi_blue} size={20} />
+ <Icon testID='circle-icon' name="check-circle" color={colors.bondi_blue} size={20} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->